index.html

```diff
@@ -3,7 +3,1043 @@
   <head>
     <meta charset="UTF-8" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
-    <title>Document</title>
+    <title>TNW</title>
+    <link href="./fontawesome/css/fontawesome.min.css" rel="stylesheet" />
+    <link href="./fontawesome/css/solid.min.css" rel="stylesheet" />
+    <link href="./fontawesome/css/brands.min.css" rel="stylesheet" />
+    <link rel="stylesheet" href="./css/index.css" />
   </head>
-  <body></body>
+
+  <body class="body">
+    <header class="header-container">
+      <div class="nav-logo">
+        <a>
+          <img alt="The Next Web" src="./image/tnw-image-logoblack.png" class="header-img" />
+        </a>
+      </div>
+
+      <div class="first-header-container">
+        <div class="first-header-ul">
+          <p class="first-header-para"><a class=" news" href="/">News</a></p>
+          <p class="first-header-para"><a href="/" class="first-header-anchor">Events ▾</a></p>
+          <p class="first-header-para">
+            <a href="#" aria-label="the-next-web" class="first-header-anchor">Business▾</a>
+          </p>
+          <p class="first-header-para">
+            <a href="#" aria-label="the-next-web" class="first-header-anchor">AMAs</a>
+          </p>
+          <p class="first-header-para">
+            <a href="#" aria-label="the-next-web" class="first-header-anchor">Spaces</a>
+          </p>
+          <p class="first-header-para">
+            <a href="#" aria-label="the-next-web" class="first-header-anchor"
+              >Terms &amp; Conditions</a
```

Comment:
Can we use `nav` element here instead of the `div`? SInce `nav` is more semantic in this context.

Reply:
I will make the adjustment right away!